assignment-client/src/entities/EntityTreeSendThread.cpp

```diff
@@ -105,6 +110,34 @@ void EntityTreeSendThread::traverseTreeAndSendContents(SharedNodePointer node, O
         uint64_t dt = usecTimestampNow() - startTime;
         std::cout << "adebug  traversal complete " << "  Q.size = " << _sendQueue.size() << "  dt = " << dt << std::endl;  // adebug
     }
+
+    // Re-add elements from previous traveral if they still need to be sent
+    while (!prevSendQueue.empty()) {
+        EntityItemPointer entity = prevSendQueue.top().getEntity();
+        prevSendQueue.pop();
+        if (entity) {
+            bool success = false;
+            AACube cube = entity->getQueryAACube(success);
+            if (success) {
+                if (_traversal.getCurrentView().cubeIntersectsKeyhole(cube)) {
+                    float renderAccuracy = calculateRenderAccuracy(_traversal.getCurrentView().getPosition(),
+                                                                   cube,
+                                                                   _traversal.getCurrentRootSizeScale(),
+                                                                   lodLevelOffset);
+
+                    // Only send entities if they are large enough to see
+                    if (renderAccuracy > 0.0f) {
+                        float priority = _conicalView.computePriority(cube);
+                        _sendQueue.push(PrioritizedEntity(entity, priority));
```

Comment:
We can skip adding the entry back onto the queue when priority is `DO_NOT_SEND`.

I would expect the priority calculation to be faster/cheaper than `calculateRenderAccuracy()` so compute it first and bail early when it is `DO_NOT_SEND`;